gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -72,26 +73,23 @@
   private final String generatorVersion;
   private final String googleCloudResourcePrefix;
   @Nullable private final Integer maxInboundMessageSize;
-
-  private InstantiatingChannelProvider(
-      ExecutorProvider executorProvider,
-      String serviceAddress,
-      int port,
-      String clientLibName,
-      String clientLibVersion,
-      String generatorName,
-      String generatorVersion,
-      String googleCloudResourcePrefix,
-      Integer maxInboundMessageSize) {
-    this.executorProvider = executorProvider;
-    this.serviceAddress = serviceAddress;
-    this.port = port;
-    this.clientLibName = clientLibName;
-    this.clientLibVersion = clientLibVersion;
-    this.generatorName = generatorName;
-    this.generatorVersion = generatorVersion;
-    this.googleCloudResourcePrefix = googleCloudResourcePrefix;
-    this.maxInboundMessageSize = maxInboundMessageSize;
+  @Nullable private final Duration keepAliveTime;
+  @Nullable private final Duration keepAliveTimeout;
+  @Nullable private final Boolean keepAliveWithoutCalls;
+
+  private InstantiatingChannelProvider(Builder builder) {
```

Comment:
So you added the fields for keep-alive, but you never pass them to the ManagedChannelBuilder.

Reply:
dammit :(

Reply:
Thank you for catching, fixed.